src/materialized/src/lib.rs

```diff
@@ -164,7 +164,16 @@ pub fn serve(mut config: Config) -> Result<Server, failure::Error> {
     let num_timely_workers = config.num_timely_workers();
 
     // Start Tokio runtime.
-    let mut runtime = tokio::runtime::Runtime::new()?;
+    let mut runtime = tokio::runtime::Builder::new()
+        .threaded_scheduler()
+        // The default thread name exceeds the Linux limit on thread name
+        // length, so pick something shorter.
+        //
+        // TODO(benesch): use `thread_name_fn` to get unique names if that
+        // lands upstream: https://github.com/tokio-rs/tokio/pull/1921.
```

Comment:
fwiw I'm not sure what kind of uniqueness we'd want, other than unique per pool (not sure if that's possible)

Reply:
I mean `tokio:work-1`, `tokio:work-2`, etc., rather than calling them all `tokio:worker` like we have to now.

Reply:
For profiling I think that's probably nice, but I've also got an in-progress metrics change that collects CPU usage by thread name (MaterializeInc/rust-prometheus#10) where I think aggregating all threads in a pool is probably the granularity/perf tradeoff you want. I'm not certain, though.